### PR TITLE
Fixes to improve readability without changing meaning

### PR DESCRIPTION
**Pull request by Kluny**
## Description
Documentation fixes. Fix comma placement, improve word order, remove unneeded words.

## Types of changes
Metabox documentation. Wordsmithing only, no changed information. 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

### CODE REVIEW

File: docs/meta-box.md [hunk 1]
```diff
@@ -52,33 +52,32 @@ Ideally, this could be done at instantiation of the editor and help simplify thi
...
+When the component renders it will store a reference to the metaboxes container and retrieve the metaboxes HTML from the prefetch location.
 
-When the post is updated, only meta boxes areas that are active will be submitted. This removes any unnecessary requests being made. No extra revisions, are created either by the meta box submissions. A Redux action will trigger on `REQUEST_POST_UPDATE` for any active meta box. See `editor/effects.js`. The `REQUEST_META_BOX_UPDATES` action will set that meta boxes' state to `isUpdating`, the `isUpdating` prop will be sent into the `MetaBoxArea` and cause a form submission.
+When the post is updated, only metabox areas that are active will be submitted. This prevents unnecessary requests. No extra revisions are created by the meta box submissions. A Redux action will trigger on `REQUEST_POST_UPDATE` for any active meta box. See `editor/effects.js`. The `REQUEST_META_BOX_UPDATES` action will set that meta box's state to `isUpdating`. The `isUpdating` prop will be sent into the `MetaBoxArea` and cause a form submission.
```
Comment (aduth) [inline docs/meta-box.md]: In all other cases, I see "Meta Box" referred to as two separate words, e.g.

https://developer.wordpress.org/plugins/metadata/custom-meta-boxes/#what-is-a-meta-box

Should this change to "metabox" not be made?

File: docs/meta-box.md [hunk 2]
```diff
@@ -52,33 +52,32 @@ Ideally, this could be done at instantiation of the editor and help simplify thi
...
 
-If the metabox area is saving, we display an updating overlay, to prevent users from changing the form values while the meta box is submitting.
+When the metabox area is saving, we display an updating overlay, to prevent users from changing the form values in mid-save.
```
Comment (aduth) [inline docs/meta-box.md]: Maybe "while a save is in progress" ?

**Review by aduth — COMMENTED** [review]
Lots of nice changes 👍 Left a few comments for you to consider.

**Comment by Kluny** [comment]
Agreed with your comments @aduth, I've updated accordingly.

**Review by aduth — APPROVED** [review]
Thanks, looks great 👍

**Pull request merged**